submariner-operator/issue-149-node-placement.md

```diff
@@ -0,0 +1,94 @@
+# Node Placement
+
+[Add support for nodeSelector and toleration scheduling for submariner pods](https://github.com/submariner-io/enhancements/issues/149)
+
+## Summary
+
+Submariner deploys many different agents as part of its core deployment. Except for Gateway, globalnet agent and
+metrics proxy, which are deployed on designated gateway nodes, there is no way for users to control or limit where rest
```

Comment:
This should mention the route agent as well, for completeness (even though it needs to be on every node).